ci-operator/step-registry/openstack/provision/machinesubnet/openstack-provision-machinesubnet-commands.sh

```diff
@@ -51,52 +58,78 @@ else
   exit 1
 fi
 
-NET_ID="$(openstack network create --format value --column id "${CLUSTER_NAME}-network")"
-echo "Created network: ${NET_ID}"
-echo $NET_ID>${SHARED_DIR}/BASTIONSUBNET_NET_ID
+BASTION_ROUTER_ID="$(openstack router create --format value --column id ${ZONES_ARGS} "${CLUSTER_NAME}-${CONFIG_TYPE}-router")"
+echo "Created bastion router: ${BASTION_ROUTER_ID}"
+echo ${BASTION_ROUTER_ID}>${SHARED_DIR}/BASTION_ROUTER_ID
+openstack router set ${BASTION_ROUTER_ID} --external-gateway ${OPENSTACK_EXTERNAL_NETWORK} >/dev/null
+echo "Connected bastion router ${BASTION_ROUTER_ID} to external network: ${OPENSTACK_EXTERNAL_NETWORK}"
```

Comment:
Should we check that `${OPENSTACK_EXTERNAL_NETWORK} != ""`?

Reply:
I don't think we need, the previous command would have failed. `OPENSTACK_EXTERNAL_NETWORK` has a value for each platform, I think this is safe to keep it that way.